config.development.yaml

```diff
@@ -2,3 +2,5 @@ api: http://127.0.0.1:3000
 buildConfig: development
 
 users: !!import/single ./config.users.yaml
+
+NOTIFY_API_KEY: initial_test_2-63998e44-a39f-4f58-b2a7-bb8f7a4269e1-2075e450-b620-4b8f-ac60-32d0427e6a44
```

Comment:
Hi @justan0therdave - we can't have api keys in the repo. I will talk you through this when you get back.